BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -83,10 +86,17 @@ @interface BOXURLSessionManager() <NSURLSessionDelegate, NSURLSessionTaskDelegat
 //during session/task's delegate callbacks, we call appropriate methods on task delegate
 @property (nonatomic, readonly, strong) NSMutableDictionary<NSString *, NSMutableDictionary<NSNumber *, BOXSessionTaskAndDelegate *> *> *backgroundSessionIdToSessionTask;
 
+//A map to associate backgroundSessionId and equivalent background NSURLSession objects that this BOXURLSessionManager knows about
+//If inside an app, there will be one background session for app, and 0 or more from previous runs of extensions
+//If inside an extension, there will be one background session for extension
 @property (nonatomic, readonly, strong) NSMutableDictionary<NSString *, NSURLSession *> *backgroundSessionIdToSession;
 
+//A cache client to cache background session task-related data to file system for retrieval after app restarts
 @property (nonatomic, readwrite, strong) BOXURLSessionCacheClient *cacheClient;
 
+//Indicate if background session's setup has completed
+@property (nonatomic, readwrite, assign) BOOL finishSettingUpBackgroundSession;
```

Comment:
Minor: Finish reads like a verb here, should this be something like 'didFinishSettingUp...'?